test/smoke_test/extentions_test/test_timescaledb.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex 
+
+psql <<EOF
```

Comment:
`psql <<EOF` and `psql -c "<>"`

Are these both the same ??

Reply:
Yes! https://www.postgresql.org/docs/current/app-psql.html